src/Filesystem.cpp

```diff
@@ -28,6 +28,12 @@ using namespace NAS2D::Exception;
 
 bool FILESYSTEM_INITIALIZED = false;
 
+enum MountPosition
+{
+	Prepend = 0,
```

Comment:
Code styling should be all-caps for enumerated values. Generally something like MOUNT_PREPEND